diff -urN aaa/gnome-speech-0.2.8/drivers/freetts/java/Makefile.am gnome-speech-0.2.8/drivers/freetts/java/Makefile.am
--- aaa/gnome-speech-0.2.8/drivers/freetts/java/Makefile.am	2003-11-25 03:09:01.000000000 +0100
+++ gnome-speech-0.2.8/drivers/freetts/java/Makefile.am	2003-11-25 20:52:00.000000000 +0100
@@ -1,7 +1,7 @@
 if BUILD_FREETTS_DRIVER
 FREETTS_DRIVER_CLASSES= \
 	org/GNOME/Speech/*.class
-CLEANFILES += $(FREETTS_DRIVER_CLASSES)
+CLEANFILES = $(FREETTS_DRIVER_CLASSES)
 CLEANFILES+=freetts-synthesis-driver.jar
 
 $(FREETTS_DRIVER_CLASSES): org/GNOME/Speech/*.java
diff -urN aaa/gnome-speech-0.2.8/drivers/freetts/native/Makefile.am gnome-speech-0.2.8/drivers/freetts/native/Makefile.am
--- aaa/gnome-speech-0.2.8/drivers/freetts/native/Makefile.am	2003-11-25 03:14:14.000000000 +0100
+++ gnome-speech-0.2.8/drivers/freetts/native/Makefile.am	2003-11-25 20:52:48.000000000 +0100
@@ -15,17 +15,13 @@
 	$(gnome_speech_CFLAGS)
 endif
 
-LDFLAGS = \
-	$(gnome_speech_LIBS) \
-	@LT_VERSION_INFO@
-
 freettsdriver.h:
 	$(JAVAH) \
 	-classpath ../java:$(GNOME_SPEECH_CLASSPATH):$(FREETTS_CLASSPATH) \
 	-o freettsdriver.h \
 	org.GNOME.Speech.FreeTTSSynthesisDriver
 
-CLEANFILES += freettsdriver.h
+CLEANFILES = freettsdriver.h
 
 libfreettsdriver_la_SOURCES = \
 	freettsdriver.c
