diff -urN gnome-speech-0.2.7.orig/configure.in gnome-speech-0.2.7/configure.in
--- gnome-speech-0.2.7.orig/configure.in	Tue Aug 26 06:53:29 2003
+++ gnome-speech-0.2.7/configure.in	Tue Aug 26 07:11:30 2003
@@ -214,7 +214,7 @@
    JAVA_SUPPORT="true"
 fi
 AC_SUBST(JAVA_SUPPORT)
-AM_CONDITIONAL(BUILD_JAVA_PACKAGE, test "x$JAVA_SUPPORT" = "xtrue")
+AM_CONDITIONAL(BUILD_JAVA_PACKAGE, no)
 
 AC_SUBST(BUILD_JAVA_PACKAGE)
 
