diff -urN aa/gnome-speech-0.3.3/drivers/freetts/java/org/GNOME/Speech/Makefile.am gnome-speech-0.3.3/drivers/freetts/java/org/GNOME/Speech/Makefile.am
--- aa/gnome-speech-0.3.3/drivers/freetts/java/org/GNOME/Speech/Makefile.am	2004-06-10 18:12:39.000000000 +0200
+++ gnome-speech-0.3.3/drivers/freetts/java/org/GNOME/Speech/Makefile.am	2004-06-10 19:42:36.632672016 +0200
@@ -1,9 +1,7 @@
 JAVA_SRC=$(wildcard *.java)
 CLASSES=$(patsubst %.java,%.class,$(JAVA_SRC))
 
-SOURCES= \
-	FreeTTSSynthesisDriver.java \
-	FreeTTSSpeaker.java
+SOURCES= FreeTTSSynthesisDriver.java FreeTTSSpeaker.java
 
 EXTRA_DIST = \
 	FreeTTSSynthesisDriver.java \
